CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased][Unreleased_log]
 --------------
+### Added
+- In Linux, 0-base enclave creation is enabled.
```

Comment:
It would be good to provide a one-line description for why someone would want to create zero-based enclaves.